index.html

```diff
@@ -154,31 +154,52 @@ <h1 class="title">
                                             <p class="subtitle">Mozilla Gujarat</p>
                                             <a href="http://mozillagujarat.org">Website</a>
                                             &nbsp;<a href="https://t.me/joinchat/CpXrs0J9FOfBVAB4TTEzAw">Telegram</a>
+                                            &nbsp;<a href="https://github.com/mozguj">GitHub</a>
                                             &nbsp;<a href="https://twitter.com/mozguj">Twitter</a>
-                                            &nbsp;<a href="https://github.com/mozguj">Github</a>
-                                            <a href="https://www.instagram.com/mozguj/">Instagram</a>
+                                            &nbsp;<a href="https://www.instagram.com/mozguj/">Instagram</a>
```

Comment:
I think adding nbsp at last link creates a spacing issue 

![Screenshot 2019-04-16 at 12 21 04 PM](https://user-images.githubusercontent.com/21317964/56188097-31230f00-6042-11e9-99c7-bdd7db3de372.png)

I don't think there is any need to add nbsp here. 
@asdofindia can you confirm and check it out?

Reply:
All of the non-breaking spaces maybe causing the same problem at different widths. It doesn't matter much either way.
![image](https://user-images.githubusercontent.com/945777/56188630-a511e700-6043-11e9-85a3-135996c83caa.png)

Reply:
I put the wrong links for Mozilla Kerala Website and the telegram chat link.f fixed those

Reply:
@obitodarky  I put the wrong links for Mozilla Kerala Website and the telegram chat link, fixed those

Reply:
@asdofindia but it can be fixed, shouldn't we fix it?  Like so
![Screenshot 2019-04-16 at 12 37 08 PM](https://user-images.githubusercontent.com/21317964/56189001-83fdc600-6044-11e9-9de0-aab2247bef46.png)


> All of the non-breaking spaces maybe causing the same problem at different widths. It doesn't matter much either way.
> ![image](https://user-images.githubusercontent.com/945777/56188630-a511e700-6043-11e9-85a3-135996c83caa.png)